app/views/accounts/_og.html.haml

```diff
@@ -0,0 +1,8 @@
+%meta{ property: 'og:site_name', content: site_title }/
+%meta{ property: 'og:type', content: defined?(type) ? type : 'website' }/
```

Comment:
Ah, I see that it gets set on on `accounts/show` but not the other two.

I might just leave this one line (type) out of the partial, and then include it in accounts/show as "profile" and leave it out of the other two views, since "website" is the default value for those.